armi/resources/mcc-nuclides.yaml

```diff
@@ -1,1239 +1,1783 @@
 # This file contains the nuclides that are defined by the MC2-2 and MC2-3
 # codes. The MC2-2 code base uses ENDF/B-V.2 and the MC2-3 code base uses
-# ENDF/B-VII.0. This file can be amended in the future for MC2-3 as the
-# code base changes, but the nuclides that MC2-3 models are consistent with
-# the data that is supplied by ENDF/B-VII.0. See: Appendix B of ANL/NE-11/41 Rev.3
+# ENDF/B-VII.0 or ENDF/B-VII.1. This file can be amended in the future for 
+# MC2-3 as the code base changes, but the nuclides that MC2-3 models are 
+# consistent with the data that is supplied by ENDF/B-VII.0. 
+# See: Appendix B of ANL/NE-11/41 Rev.3 for V.2 and VII.0 isotopes
 # Public Link: https://publications.anl.gov/anlpubs/2018/10/147840.pdf.
 AC225:
   ENDF/B-V.2: null
   ENDF/B-VII.0: AC2257
+  ENDF/B-VII.1: AC2257
```

Comment:
@aaronjamesreynolds I'm not the right person to validate this data.  If you were to pick something to validate this file for correctness, who would you pick?

Reply:
Okay, I did a review that these strings were in the data file you pointed me towards.

So, great, that was fun.